### PR TITLE
XW-5217 | Top Articles recirculation

### PR DESCRIPTION
**Pull request by rybmat**
## Links

* http://wikia-inc.atlassian.net/browse/XW-5217
* https://github.com/Wikia/app/pull/15931

@Wikia/x-wing

### CODE REVIEW

File: app/templates/components/recirculation-prefooter.hbs [hunk 1]
```diff
@@ -2,6 +2,26 @@
 	<div class="recirculation-prefooter__header">
 		<h2 class="wds-font-size-base wds-font-weight-bold wds-text-transform-uppercase wds-midlight-navy">{{i18n 'recirculation.trending-fandom-articles'}}</h2>
 	</div>
+{{/if}}
+
+{{#if topArticles.length}}
+	<div class="recirculation-prefooter__wiki-articles">
+		<header class="recirculation-prefooter__wiki-articles-header wds-leading-normal wds-font-size-base wds-font-weight-bold wds-text-transform-uppercase">
+
```
Comment (hakubo) [inline app/templates/components/recirculation-prefooter.hbs]: unnecessary \n?

File: app/components/recirculation-prefooter.js [hunk 2]
```diff
@@ -1,13 +1,15 @@
 import { defer } from 'rsvp';
-import fetch from 'fetch';
 import { inject as service } from '@ember/service';
 import Component from '@ember/component';
 import { computed } from '@ember/object';
+import { reads, not, equal, and, bool } from '@ember/object/computed';
```
Comment (xkxd) [inline app/components/recirculation-prefooter.js]: `not` and `bool` are not used

File: app/controllers/article.js [hunk 3]
```diff
@@ -11,10 +11,7 @@ export default Controller.extend(WikiPageControllerMixin, {
   wikiVariables: service(),
 
   commentsPage: alias('application.commentsPage'),
-  loadExternals: not('application.noExternals'),
```
Comment (xkxd) [inline app/controllers/article.js:16]: please remove imports as well

File: app/templates/article.hbs [hunk 4]
```diff
@@ -7,9 +7,9 @@
 		toggleSiteHeadShadow=(action 'toggleSiteHeadShadow' target=application)
 	}}
 </div>
-{{#if displayRecirculation}}
-	{{recirculation-prefooter}}
-{{/if}}
+{{recirculation-prefooter
+		applicationWrapperVisible=applicationWrapperVisible
```
Comment (xkxd) [inline app/templates/article.hbs]: indent

**Review by xkxd — APPROVED** [review]

File: app/controllers/article.js (hunk 3 above)
Comment (rybmat) [reply app/controllers/article.js:16]: done

**Pull request merged**